.github/workflows/benchmarks.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           name: build-output
           path: preact.tgz
       - name: Initialize Tachometer Results Report
-        uses: andrewiggins/tachometer-reporter-action@v2
+        uses: andrewiggins/tachometer-reporter-action@v2.3.0
```

Comment:
Gonna update this to "@v2" once I verify this works and publish it to the v2 tag

Reply:
Done!